Q/QuantumEspresso/build_tarballs.jl

```diff
@@ -4,28 +4,19 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "QuantumEspresso"
-version = v"7.0.1"
-quantumespresso_version = v"7.0.0"
-
-# ES 2022-08-01: I tried updating to 7.1, but I encountered build problems:
-# - v7.1 encounters ICEs in gfortran; requires at least GCC 10
-# - the build fails because the directory "../W90" is not found at some point
-# version = v"7.1"
-# quantumespresso_version = v"7.1"
+version = v"7.4.0"
```

Comment:
```suggestion
version = v"7.4.1"
```
why don't follow the upstream version here?